### DISCUSSION TITLE
3D maps on older devices?

### DISCUSSION THREAD

**Discussion by jrzelazny** [osmandapp/OsmAnd#18811] [Maps Related]
I downloaded the most recent OsmAnd+ update (4.6.9) to my three devices. I am very interested in the relatively new 3D relief maps.

The 3D maps seem to work fine on my Galaxy S21 Ultra, Android 13. It works on my Galaxy Tab S6, Android 12. But the 3D maps don't work on my Kyocery Duraforce Pro 2, Android 10.

When I go into the "Configure Maps" menu, there is no "Topography" option as there are on the more recent devices, even though I have the 3D Relief maps downloaded to the device.

Is there a way to activate this, or is this simply a limitation of Android 10 and older devices, that they don't support the 3D maps?

**Comment by mikehgentry** [comment]
It'll only work on OpenGL mode, which is in OsmAnd Settings. 

https://osmand.net/docs/user/personal/global-settings/#map-rendering-engine

**Comment by jrzelazny** [comment]
I had it set on OpenGL mode.

More playing around and I realized I didn't have the "Topography" Plugin turned on. I thought that would fix it, but still nothing.

If I go into the "Configure Map" menu, now the "Topography" dropdown is there, but it doesn't have any entries. That is, if I tap on it, nothing happens.

However, I found that if I tap on the "Map rendering" dropdown, it opens to a big list of options, and at the very bottom is the option to turn on "3D Relief." Unlike my other devices where that option is under the "Topography" menu. I don't know why it's different, but the 3D maps now work.

**Reply by DmitryAlexei** [reply]
You have probably modified the Configure map menu on the last device. Try it with the default profile.
